roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -222,6 +227,8 @@ class RooDataHist : public RooAbsData, public RooDirItem {
   friend class RooAbsCachedPdf ;
   friend class RooAbsCachedReal ;
   friend class RooDataHistSliceIter ;
+  // for access into copied dataset:
+  friend class RooFit::TestStatistics::RooAbsL;
```

Comment:
The changes in this file are not necessary, `RooAbsL` is not using any non-public functions in RooDataHist.

Reply:
Ah, weird. I thought this was necessary for RooAbsL::initClones, but maybe I've edited the dependency out.